pkg/appgw/certificates_test.go

```diff
@@ -0,0 +1,120 @@
+package appgw
+
+import (
+	"testing"
+
+	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/annotations"
+	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/k8scontext"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/client-go/tools/cache"
+)
+
+func TestMakeHostToSecretMap(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Test setting up SSL redirect annotations")
+}
+
+var _ = Describe("makeHostToSecretMap", func() {
+	Context("looking at TLS specs", func() {
```

Comment:
This context seems odd? Maybe :
` Testing caching of secrets for ingress with TLS spec`?